tests/mapped_struct.py

```diff
@@ -1699,6 +1706,39 @@ def testSubsetSuperset(self):
         self.assertTrue(small < big)
         self.assertTrue(big > small)
 
+        a = self.pack([1., 2.])
+        b = self.pack([1., 2., 3., 4.])
+        self.assertTrue(a < b)
+        self.assertTrue(a <= b)
+        self.assertFalse(b < a)
+        self.assertFalse(b <= a)
+        self.assertNotEquals(a, b)
+
+        a = self.pack([1., 2., 3.])
+        b = self.pack([1., 2.5, 3., 4.])
+        self.assertFalse(a < b)
+        self.assertFalse(a <= b)
+        self.assertFalse(b < a)
+        self.assertFalse(b <= a)
+        self.assertNotEquals(a, b)
+
+        a = self.pack([1., 2.5, 3.])
+        b = self.pack([1., 2., 3., 4.])
+        self.assertFalse(a < b)
+        self.assertFalse(a <= b)
+        self.assertFalse(b < a)
+        self.assertFalse(b <= a)
+        self.assertNotEquals(a, b)
+
+        a = self.pack([1., 2., 3., 10.])
+        b = self.pack([1., 2., 3., 4.])
+        self.assertFalse(a < b)
+        self.assertFalse(a <= b)
+        self.assertFalse(b < a)
+        self.assertFalse(b <= a)
+        self.assertNotEquals(a, b)
```

Comment:
why do we have to add all this asserts after migrating to python3? the behavior shouldn't be the same?